frontend/src/app-components/attachment/AttachmentUploader.tsx

```diff
@@ -78,12 +80,15 @@ const AttachmentUploader: FC<FileUploadProps> = ({
   );
   const { t } = useTranslate();
   const [isDragOver, setIsDragOver] = useState<boolean>(false);
+  const [isUploading, setIsUploading] = useState<boolean>(false);
   const { toast } = useToast();
   const { mutateAsync: uploadAttachment } = useUpload(EntityType.ATTACHMENT, {
```

Comment:
You don't have to add an extra state :
```suggestion
  const { toast } = useToast();
  const { mutateAsync: uploadAttachment, isLoading } = useUpload(EntityType.ATTACHMENT, {
```

Reply:
100%, as mentioned we can use useQuery isLoading state.
The the recommendation are implemented in a new [PR](https://github.com/Hexastack/Hexabot/pull/920)